src/mds/MDSRank.h

```diff
@@ -577,6 +580,13 @@ class MDSRank {
 private:
     mono_time starttime = mono_clock::zero();
 
+    // "task" string that gets displayed in ceph status
+    const std::string SCRUB_STATUS_KEY = "scrub status";
```

Comment:
`inline static const std::string SCRUB_STATUS_KEY = "scrub status";` would be better.

Reply:
See also: https://en.cppreference.com/w/cpp/language/static

Reply:
ack